libraries/radspreadstreamprocessing/overview.md

```diff
@@ -12,13 +12,40 @@ position: 0
 
 Thank you for choosing Telerik **RadSpreadStreamProcessing**!
 
-**RadSpreadStreamProcessing** is a library that allows you create big spreadsheet documents and export them to the most common file formats with a great performance and minimal resources.
+This article briefly explains the Spread Stream library specifics as what is spread streaming, how it works compared to the **RadSpreadProcessing** library and when to use it.
 
 ![](images/SpreadStreamProcessing-Overview_01.png)
 
+## What is Spread Streaming?
+
+Spread streaming is a document processing paradigm that allows you to create big spreadsheet documents with great performance and minimal memory footprint. 
+
+The key for the memory efficiency is that the spread streaming library writes the spreadsheet content directly to a stream without creating and preserving the spreadsheet document model in memory. Each time a value is set, the export kicks-in and the new value is written into the stream. This allows you to create large documents with an excellent performance.
```

Comment:
This is not absolutely true. All the values that should be preserved in the exported document are written when the corresponding object is disposed. E.g., if the users set a value to a cell, this value along with the other cell properties will be written when the used ICellExporter object is disposed.